nmpolicy/internal/resolver/filter.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 func filter(inputState map[string]interface{}, path ast.VariadicOperator, expectedNode ast.Node) (map[string]interface{}, error) {
-	filtered, err := applyFuncOnPath(inputState, path, expectedNode, mapContainsValue, true)
+	filtered, err := applyFuncOnMap(path, inputState, expectedNode, mapContainsValue, true, true)
```

Comment:
I see `applyFuncOnMap` as an internal method that should be used only by `applyFuncOnPath`.
Anyway, how is this change related to the commit/PR?

Reply:
this is internal code, and we know that inputState is a map, does not make sense to call `applyFuncOnPath`, this way we remove one call.